specification/2.0/README.md

```diff
@@ -831,57 +832,108 @@ A skin is instanced within a node using a combination of the node's `mesh` and `
+Any colorspace information (such as ICC profiles, intents, etc) from PNG or JPEG containers must be ignored.
+
+> **Implementation Note**: This increases portability of an asset, since not all image decoding libraries fully support custom color conversions. To achieve correct rendering, WebGL runtimes must disable such conversions by setting `UNPACK_COLORSPACE_CONVERSION_WEBGL` flag to `NONE`.
+
+In the following example asset provides two versions of the same image: in sRGB and in linear colorspaces; so clients with hardware sRGB filtering support (or with enough fragment shader processing power) benefit from reduced banding, whilst low-power clients still get correct texel values by using linear-space image data directly.
 
 ```json
 {
     "images": [
         {
-            "uri": "duckCM.png"
-        },
-        {
-            "bufferView": 14,
-            "mimeType": "image/jpeg" 
+            "sources": [
+                {
+                    "internalformat": 32856,
+                    "mimeType": "image/png",
+                    "uris": ["duck_diffuse_linear.png"]
+                },
+                {
+                    "internalformat": 35907,
+                    "mimeType": "image/png",
+                    "uris": ["duck_diffuse_srgb.png"]
+                }
+            ]
         }
     ]
 }
 ```
-> **Implementation Note**: With WebGL API, the first pixel transferred from the `TexImageSource` (i.e., HTML Image object) to the WebGL implementation corresponds to the upper left corner of the source. Non-WebGL runtimes may need to flip Y axis to achieve correct texture rendering.
 
-### Samplers
+#### Image Formats
+
+glTF 2.0 allows a limited set of possible `internalformat` values based on OpenGL ES 3.0 enums.
+ 
+Each `internalformat` value defines exactly one matching pair of `type` and `format` parameters of OpenGL ES 2.0/3.0 texture specification. See table below for details:
+
+| `source.internalformat` | OpenGL ES 2.0 Format & Internal Format | OpenGL ES 3.0 Format | OpenGL ES 3.0 Internal Format | OpenGL ES 2.0/3.0 Type |
+|-------------------------|----------------------------------------|----------------------|-------------------------------|---------------------------|
+| ALPHA (`6406`) | ALPHA | ALPHA | ALPHA | UNSIGNED_BYTE |
+| LUMINANCE (`6409`) | LUMINANCE | LUMINANCE | LUMINANCE | UNSIGNED_BYTE |
+| LUMINANCE_ALPHA (`6410`) | LUMINANCE_ALPHA | LUMINANCE_ALPHA | LUMINANCE_ALPHA | UNSIGNED_BYTE |
+| RGB8 (`32849`) | RGB | RGB | RGB8 | UNSIGNED_BYTE |
+| RGBA4 (`32854`) | RGBA | RGBA | RGBA4 | UNSIGNED_SHORT_4_4_4_4 |
+| RGB5_A1 (`32855`) | RGBA | RGBA | RGB5_A1 | UNSIGNED_SHORT_5_5_5_1 |
+| RGBA8 (`32856`) | RGBA | RGBA | RGBA8 | UNSIGNED_BYTE |
+| SRGB8 (`35905`) | SRGB_EXT | RGB | SRGB8 | UNSIGNED_BYTE |
```

Comment:
Does this require an extension? https://www.khronos.org/registry/webgl/extensions/EXT_sRGB/

The spec should clarify this.